src/tgftools/analysis.py

```diff
@@ -656,11 +656,11 @@ def get_partner(self) -> pd.DataFrame:
     def get_gp(self) -> pd.DataFrame:
         """Returns data-frame of the GP elements that are needed for reporting."""
 
-        if self.disease_name != 'HIV':
+        if self.disease_name != 'test':
             gp_data = self.database.gp.df['central'].unstack()
         else:
             # Get GP for HIV
-            gp_data = self.portfolio_projection_counterfactual('GP_GP')  # todo softcode
+            gp_data = self.portfolio_projection_counterfactual('GP')  # todo softcode
```

Comment:
this is a change to the framework in `analysis.py`, so is going to have implications for running IC7. We want to avoid changes in these files, and that's why we want to "soft code" the scenario that is used for the counterfatual, per the 'todo' comment.